map/src/test/java/openlr/map/utils/BearingPointCalculatorTest.java

```diff
@@ -0,0 +1,63 @@
+package openlr.map.utils;
+
+import openlr.map.GeoCoordinates;
+import openlr.map.Line;
+import openlr.map.MapDatabase;
+import openlr.map.teststubs.OpenLRMapDatabaseAdaptor;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class BearingPointCalculatorTest {
```

Comment:
We need a test case for situations where you have to walk several lines until you find the right one to calculate the projection.

Reply:
0975cd3318040870779bc3de4b706cd2ffb8b2ef